source/geometry_model/chunk.cc

```diff
@@ -99,7 +147,7 @@ namespace aspect
             output_vertex[1] = std::atan2(v[1], v[0]);
             // See 2D case
             if (output_vertex[1] < 0.0)
-              if (output_vertex[1] < point1_lon - std::abs(point1_lon)*std::numeric_limits<double>::epsilon())
+              if (output_vertex[1] < point1_lon - 100 * std::abs(point1_lon)*std::numeric_limits<double>::epsilon())
```

Comment:
Why 100? is that necessary? If so describe in a comment why, if not make it the same as in the other function.